Dockerfile

```
@@ -0,0 +1,10 @@
+FROM alpine:3.13 as download
+
+RUN wget -O sonarqube-community-branch-plugin.jar https://github.com/mc1arke/sonarqube-community-branch-plugin/releases/download/1.3.2/sonarqube-community-branch-plugin-1.3.2.jar
```

Now that I think about that, it might be beneficial to avoid relying on a pre-built JAR and just build the plugin ourselves. Here is how I would go about doing it:
```suggestion
FROM openjdk:11-jdk as build
ARG PLUGIN_RELEASE_REF=1.3.2

RUN git clone --depth 1 https://github.com/mc1arke/sonarqube-community-branch-plugin.git -b $PLUGIN_RELEASE_REF && \
    cd sonarqube-community-branch-plugin && \
    ./gradlew --no-daemon build -x test && \
    mv build/libs/sonarqube-community-branch-plugin*.jar ../plugin.jar
```